app/src/main/assets/web_extensions/webcompat_youtube/manifest.json

```diff
@@ -18,5 +18,10 @@
       "run_at": "document_start",
       "all_frames": true
     }
-  ]
+  ],
+  "permissions": ["webRequest", "webRequestBlocking", "*://*.youtube.com/*"],
```

Comment:
You'll also need to include `"*://*.youtube-nocookie.com/*"` here.

Reply:
good catch! fixed